### PR TITLE
Fix lack of feedback when users try to reach admins

### PR DESCRIPTION
**Pull request by optimumtact**
This specifically reduces issues with players getting frustrated when they adminhelp to an AFK admin.

Players can also see in adminwho the afk admins

### CODE REVIEW

File: code/modules/admin/verbs/adminhelp.dm [hunk 1]
```diff
@@ -126,6 +126,7 @@
 	//send it to irc if nobody is on and tell us how many were on
 	var/admin_number_present = send2irc_adminless_only(ckey,original_msg)
 	log_admin("HELP: [key_name(src)]: [original_msg] - heard by [admin_number_present] non-AFK admins who have +BAN.")
+	src << "<span class='notice'>Your admin help was recieved by [admin_number_present] non-AFK admins</span>"
```
Comment (bgobandit) [inline code/modules/admin/verbs/adminhelp.dm]: >recieved

Comment (optimumtact) [inline code/modules/admin/verbs/adminhelp.dm]: That's the correct way to spell it in new zealand

**Comment by MrStonedOne** [comment]
See, I was thinking just removing afk admins from adminwho.

File: code/modules/client/verbs/who.dm [hunk 2]
```diff
@@ -80,6 +80,8 @@
 		for(var/client/C in admins)
 			if(!C.holder.fakekey)
 				msg += "\t[C] is a [C.holder.rank]\n"
+			if(C.is_afk())
+				msg += " (AFK)"
```
Comment (MrStonedOne) [inline code/modules/client/verbs/who.dm]: This needs to be tabbed one level, otherwise afk stealth mins would show an empty (AFK) line

Comment (MrStonedOne) [inline code/modules/client/verbs/who.dm]: or reverse the above check and make it an early ~~return~~ continue.

Comment (MrStonedOne) [inline code/modules/client/verbs/who.dm]: Also, it adds \n to the end of the line, so the ` (AFK)` bit would end up on the next line.

File: code/modules/admin/verbs/adminhelp.dm (hunk 1 above)
Comment (Cyberboss) [inline code/modules/admin/verbs/adminhelp.dm]: http://recieved.co.uk/

Comment (Tacolizard) [inline code/modules/admin/verbs/adminhelp.dm]: >>recieved

Comment (Mervill) [inline code/modules/admin/verbs/adminhelp.dm]: Honour

**Comment by lzimann** [comment]
Removing the AFK admin from adminwho is also a nice solution to this. Both solutions are decent regardless.

**Comment by optimumtact** [comment]
Comments addressed, this is ready for review

File: code/modules/admin/verbs/adminhelp.dm [hunk 3]
```diff
@@ -126,6 +126,8 @@
 	//send it to irc if nobody is on and tell us how many were on
 	var/admin_number_present = send2irc_adminless_only(ckey,original_msg)
 	log_admin("HELP: [key_name(src)]: [original_msg] - heard by [admin_number_present] non-AFK admins who have +BAN.")
+	if(admin_number_present <= 0)
+		src << "<span class='notice'>No active admins are online, your adminhelp was sent to the admin irc</span>"
```
Comment (AnturK) [inline code/modules/admin/verbs/adminhelp.dm]: Mind adding the . at the end.

**Comment by optimumtact** [comment]
Damn english grammar terrorists

**Comment by lzimann** [comment]
dot that shit up fam

**Pull request merged**